website/docs/d/is_backup_policies.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : is_backup_policies"
+description: |-
+  Get information about backup policies.
+---
+
+# ibm_is_backup_policies
+
+Provides a read-only data source for BackupPolicyCollection. For more information, about backup policy in your IBM Cloud VPC, see [Backup policy](https://cloud.ibm.com/docs/vpc?topic=vpc-creating-backup-policy).
```

Comment:
We need to verify link for all the documents

Reply:
modified